activation/poetdb.go

```diff
@@ -19,19 +21,58 @@ import (
 	"github.com/spacemeshos/go-spacemesh/sql/poets"
 )
 
+// PoetDbOptions are options for PoetDb.
+type PoetDbOptions struct {
+	cacheSize int
+}
+
+type PoetDbOption func(*PoetDbOptions)
+
+// WithCacheSize sets the cache size for PoetDb.
+func WithCacheSize(size int) PoetDbOption {
```

Comment:
It should probably be used in `node.go` with the `config.PoetProofsCache`, shouldn't it?

Reply:
yes thank you for catching